app/views/admin/creator_settings/_form.html.erb

```diff
@@ -52,11 +54,11 @@
     <legend class="crayons-field__label mb-2">Who can join this community?</legend>
     <div>
       <div class="mb-2">
-        <%= radio_button_tag :invite_only_mode, "0", class: "crayons-field crayons-field--radio", required: true %>
```

Comment:
The `crayons-radio` styles allow the radio buttons to receive the brand color styling. In addition, the required fields weren't being applied here.